Lib/test/test_asyncio/test_selector_events.py

```diff
@@ -547,6 +547,22 @@ def test_pause_resume_reading(self):
         self.assertFalse(tr.is_reading())
         self.loop.assert_no_reader(7)
 
+    def test_pause_reading_connection_made(self):
```

Comment:
Could you add a test where `pause_reading()` is called from *inside* `connection_made()`? That's what gh-103607 reports is broken.